tests/end_to_end/test_slack_integration/data/configuration/slack_api_script.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+
+import argparse
+import requests
+from http import HTTPStatus
+
+
+def get_parameters():
+    """
+    Returns:
+        argparse.Namespace: Object with the user parameters.
+    """
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--token', '-t', type=str, action='store', required=True)
+    parser.add_argument('--channel', '-c', type=str, action='store', required=True)
+    parser.add_argument('--messages_limit', '-m', type=int, action='store', required=True)
+    parser.add_argument('--path', '-p', type=str, action='store', required=True)
+
+    arguments = parser.parse_args()
+
+    return arguments
+
+
+def main():
+    parameters = get_parameters()
+
```

Comment:
Add comments during the different sections

Reply:
Done in 05475bc